src/libraries/Native/Unix/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -79,9 +80,9 @@ NetSecurityNative_ImportUserName(uint32_t* minorStatus, char* inputName, uint32_
 Shims the gss_import_name method with nametype = GSS_C_NT_USER_NAME.
 */
 PALEXPORT uint32_t NetSecurityNative_ImportPrincipalName(uint32_t* minorStatus,
-                                                         char* inputName,
-                                                         uint32_t inputNameLen,
-                                                         GssName** outputName);
+    char* inputName,
```

Comment:
Undo changes in this file?

Reply:
I've tried and it caused build failures. I am not sure why it is ok to not have this include in libs. Let me take a look.

I vaguely remember that the next #if will not be able to choose which way to go.
It could be just different compiler settings about tolerance to missing defines.

Reply:
@janvorli - I may need help here. 
As I understand how configs work, they will result in an .h file generated which captures various checks in a form of defines. So why could it be ok not to include the file when building libs partition?

Reply:
I am not sure I understand the response. Aren't these just formatting changes?

Reply:
Formatting changes are accidental . VS keep trying to reformat this file. I will revert the whitespace changes. 
There is also a new include - `#include "pal_config.h"`.   I thought the initial comment about reverting that.

I remember trying to revert that and it caused errors. I will try reverting it again, so we could see what kind of errors come from it.

Reply:
I was referring to the response to @jkotas's suggestion.

Reply:
I have missed that added include. Maybe @jkotas missed it too?